Client/Application/Delegates/SceneDelegate.swift

```diff
@@ -65,7 +65,7 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
             SceneDelegate.shouldHandleUrpLookup = false
             
             if let urp = UserReferralProgram.shared {
-                browserViewController.handleReferralLookup(urp, checkClipboard: false)
+                browserViewController.handleReferralLookup(urp)
```

Comment:
Should this still be here?

Reply:
yes, we have to do the lookup but without using clipboard, this is now the only place where we call this method